src/common/k8s-api/kube-api.ts

```diff
@@ -542,6 +571,33 @@ export class KubeApi<T extends KubeObject> {
 
     logger.info(`[KUBE-API] watch (${watchId}) ${retry === true ? "retried" : "started"} ${watchUrl}`);
 
+    if (!this.watchingSystemStatus) {
+      this.watchSystemStatus();
+    }
+
+    onceSystemSuspend().then(async (suspended) => {
+      if (suspended) {
+        logger.info(`[KUBE-API] system suspended, abort watching of ${watchUrl}...`);
+
+        try {
+          abort();
```

Comment:
Got this when calling `abort()`, not sure why...
```
[0] error:   ┏ [KUBE-API] error aborting watch (devcluster-1) Expected 'this' to be an 'AbortController' object, but got undefined +0ms
[0] error:   ┃ [1] TypeError: Expected 'this' to be an 'AbortController' object, but got undefined
[0] error:   ┃ [2]     at getSignal (webpack:///./node_modules/abort-controller/dist/abort-controller.js?:104:15)
[0] error:   ┃ [3]     at abort (webpack:///./node_modules/abort-controller/dist/abort-controller.js?:91:21)
[0] error:   ┃ [4]     at KubeApi.eval (webpack-internal:///./src/common/k8s-api/kube-api.ts:450:11)
[0] error:   ┃ [5]     at Generator.next (<anonymous>)
[0] error:   ┃ [6]     at eval (webpack-internal:///./src/common/k8s-api/kube-api.ts:68:61)
[0] error:   ┃ [7]     at new Promise (<anonymous>)
[0] error:   ┃ [8]     at __async (webpack-internal:///./src/common/k8s-api/kube-api.ts:52:10)
[0] error:   ┗ [9]     at eval (webpack-internal:///./src/common/k8s-api/kube-api.ts:446:45)
```

Reply:
Fix it by using `node-abort-controller` in extension.

Reply:
I think the `this` reference is wrong because the abort property is extracted from the class instance. In my related PR (https://github.com/lensapp/lens/pull/4388)  I just call the method using `abortController` so the `this` is correct.

Reply:
I suggest fixing it in the same way in this PR too (or just wait until the other PR is merged and fix conlifcts).